### PR TITLE
Fix sign screen

### PR DESCRIPTION
**Pull request by mazmassa**
This is related to:  https://github.com/massalabs/station-massa-wallet/issues/636#issuecomment-1710620514 and https://github.com/massalabs/station-massa-wallet/issues/624#issuecomment-1710150919

We are handling two main issue in this MR:

- add amount format to be in the standard
- rework sing a message box

### CODE REVIEW

**Review by Thykof — APPROVED** [review]

File: wails-frontend/src/pages/PasswordPromptHandler/Sign.tsx [hunk 1]
```diff
@@ -83,10 +83,16 @@ export function Sign() {
     save(e);
   }
 
+  function _getTitle(operation: string | undefined) {
```
[the remark this answers is not in the document]
Comment (mazmassa) [reply wails-frontend/src/pages/PasswordPromptHandler/Sign.tsx:94]: Convention in JS to say that is private

File: wails-frontend/src/pages/PasswordPromptHandler/Sign.tsx [hunk 2]
```diff
@@ -83,10 +83,16 @@ export function Sign() {
...
+    if(operation === 'Plain Text') return Intl.t('password-prompt.title.sign-message');
```
[the remark this answers is not in the document]
Comment (mazmassa) [reply wails-frontend/src/pages/PasswordPromptHandler/Sign.tsx]: Added in the last commit! ;)

**Review by gregLibert — APPROVED** [review]

**Comment by gregLibert** [comment]
@mazmassa to avoid code duplication in plaintext and callsc file you could use:
```jsx
export function DescriptionAccordion({ description }) {
  return (
    <div className="flex flex-col w-full h-fit">
      <AccordionCategory
        isChild={false}
        iconOpen={<FiChevronDown />}
        iconClose={<FiChevronUp />}
        customClass={'!p-0'}
        categoryTitle={
          <div className="flex items-center w-full gap-4">
            <FiInfo size={18} />
            <p>{Intl.t('password-prompt.sign.description')}</p>
          </div>
        }
      >
        <AccordionContent customClass={'px-0 pt-4 pb-0'}>
          <div className="max-w-full overflow-hidden">
            <p>{description}</p>
          </div>
        </AccordionContent>
      </AccordionCategory>
    </div>
  );
}
```
and
```jsx
function setWindowSize(additionalHeight) {
  const winWidth = 460;
  const winHeight = 460 + additionalHeight;
  WindowSetSize(winWidth, winHeight);
}
```

**Comment by mazmassa** [comment]
Sure! 

Already note here a revamp on all that popup screens anyway. We shouldn't use neither the switch/case as we did. And there's no need neither for <></> in each component.

Just haven't did all now to not break...

**Comment by gregLibert** [comment]
> Sure!
> 
> Already note here a revamp on all that popup screens anyway. We shouldn't use neither the switch/case as we did. And there's no need neither for <></> in each component.
> 
> Just haven't touched now to avoid breaks...

perfect! my comment was just related to the failure of sonarcloud CI and it's corresponding [comment]( https://github.com/massalabs/station-massa-wallet/pull/682#issuecomment-1711392713)

**Pull request merged**